folium/folium.py

```diff
@@ -948,6 +950,40 @@ def json_style(style_cnt, line_color, line_weight, line_opacity,
         self.template_vars.setdefault('geo_styles', []).append(style)
         self.template_vars.setdefault('gjson_layers', []).append(layer)
 
+    @iter_obj('image_overlay')
+    def image_overlay(self, data, image_name=None, filename='data.png',
+                      lower_left=[-90,-180], upper_right=[90,180], opacity=0.25):
+        """Simple image overlay of raster data from URL or numpy array.
+
+        """
+
+        if isinstance(data, str):
+            pass
+        try:
+            imageio.imwrite(filename, data)
+        except:
+            raise ValueError('Data is not proper image format (NxM, NxMx3 or NxMx4)')
+
+        if image_name not in self.added_layers:
+            if image_name is None:
+                image_name = data
+            else:
+                image_name = image_name.replace(" ", "_")
```

Comment:
Have to do this more that I would like.... maybe it is time for a normalization function...  do not worry about this in this PR.